kratos/processes/simple_mortar_mapper_process.h

```diff
@@ -0,0 +1,474 @@
+                        const bounded_matrix<double, TNumNodes, TNumNodes> inv_d_operator = (std::abs(aux_det) < tolerance) ? ZeroMatrix(TNumNodes) : MathUtils<double>::InvertMatrix<TNumNodes>(normalized_D, aux_det, tolerance);
+                        const bounded_matrix<double, TNumNodes, TNumNodes> p_operator = 1.0/norm_D * prod(inv_d_operator, rThisMortarConditionMatrices.MOperator); 
+                        
+                        Matrix var_origin_matrix;
+                        MortarUtilities::MatrixValue<TVarType, THist>(slave_geometry, mDestinyVariable, var_origin_matrix);
+        
+                        const Matrix var_destiny_matrix = (master_area/total_master_area) * prod(p_operator, var_origin_matrix);
+                        MortarUtilities::AddValue<TVarType, THist>(master_geometry, mDestinyVariable, var_destiny_matrix);
+                    }
+                }
+            }
+        }
+        
+        KRATOS_CATCH("");
+    }
+
+    /**
+     * This method sets both variables (origin and destination) with the same variable
+     */
+    void SetVariable(TVarType ThisVariable)
+    {
+        mOriginVariable = ThisVariable;
+        mDestinyVariable = ThisVariable;
+    }
+    
+    /**
+     * This method sets both variables (origin and destination) in a separated way
+     */
+    void SetVariables(
+        TVarType OriginVariable,
+        TVarType DestinyVariable
+        )
+    {
+        mOriginVariable = OriginVariable;
+        mDestinyVariable = DestinyVariable;
+    }
+    
+    ///@}
+    ///@name Access
+    ///@{
+
+
+    ///@}
+    ///@name Inquiry
+    ///@{
+
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    std::string Info() const override
+    {
+        return "SimpleMortarMapperProcess";
+    }
+
+    /// Print information about this object.
+    void PrintInfo(std::ostream& rOStream) const override
+    {
+        rOStream << "SimpleMortarMapperProcess";
+    }
+
+    /// Print object's data.
+    void PrintData(std::ostream& rOStream) const override
+    {
+    }
+
+    ///@}
+    ///@name Friends
+    ///@{
+
+    ///@}
+protected:
+
+    ///@name Protected static Member Variables
+    ///@{
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+    ///@}
+    ///@name Protected Inquiry
+    ///@{
+
+    ///@}
+    ///@name Protected LifeCycle
+    ///@{
+    ///@}
+
+private:
+    ///@name Static Member Variables
+    ///@{
+    ///@}
+    ///@name Member Variables
+    ///@{
+    
+    ModelPart& mrThisModelPart; // The model part to compute
+    TVarType mOriginVariable;   // The origin variable to map
+    TVarType mDestinyVariable;  // The destiny variable to map
```

Comment:
Small, I usually call it Destination. In case you want to be consistent

Reply:
Ok